### ISSUE TITLE
Compilation issue: 0.29.1.tar.gz missing crawl-ref/source/util/release_ver

### ISSUE THREAD

**Issue by escondida** [crawl/crawl#2764]
**PLATFORM and OS or BROWSER:**

Archlinux

**DCSS VERSION AND GAME INFORMATION:**

0.29.1

**DESCRIPTION:**

Can't complete compilation, because crawl-ref/source/util/release_ver is missing. As a workaround, the file is easy to create, but perhaps it should be mentioned in the release notes.

**EXTRA FILES:**

N/A

**Comment by rawlins** [comment]
> 0.29.1.tar.gz missing crawl-ref/source/util/release_ver

To build from source, you can't use a bare repository download, which is what this file is (github autogenerates these).

Non-packagers building from source are recommended to use the git repository as described in [INSTALL.md](https://github.com/crawl/crawl/blob/master/crawl-ref/INSTALL.md).

For 0.29.1, anyone packaging crawl should use: https://github.com/crawl/crawl/releases/download/0.29.1-debian/crawl_0.29.1.orig.tar.xz. (Previously this would have been labeled with `nodeps` in the filename, but the current CI scripts rename it.)

**Comment by escondida** [comment]
Ah, interesting. I am packaging crawl; I just didn't realize I needed to use the debian version. After creating the version file, compilation proceeded as normal; is there a reason not to use the repository snapshot tarball, besides lack of the version file?

**Comment by rawlins** [comment]
>is there a reason not to use the repository snapshot tarball, besides lack of the version file?

Depends on how exactly you're doing it, but the `nodeps` package is designed for linux packaging -- strips out a bunch of unnecessary files, all the contribs (since most packagers want to use pkg-config + their own packages for sdl, etc), and sets the version. That file I linked isn't debian specific, and previous was released as e.g. https://github.com/crawl/crawl/releases/download/0.29.0/stone_soup-0.29.0-nodeps.tar.xz. (We'll probably return to this naming scheme for future releases.)
